unlock-js/package.json

```diff
@@ -29,14 +29,15 @@
     }
   },
   "devDependencies": {
-    "jest": "^24.1.0",
-    "nock": "^10.0.5",
     "@babel/cli": "^7.2.3",
     "@babel/core": "^7.3.4",
     "@babel/preset-env": "^7.4.2",
     "babel-jest": "^24.7.1",
+    "jest": "^24.1.0",
```

Comment:
Not sure why npm re-ordered things :/

Reply:
they were probably done by hand before (by me...) and now they are alphabetical